README.md

```diff
@@ -1,2 +1,21 @@
-# news-for-maintainers
-Announcements of changes impacting collection contributors and maintainers
+# News for Maintainers
+Announcements of changes impacting Ansible collection contributors and maintainers.
+
+This repository is created in accordance with the [Community decision](https://github.com/ansible-community/community-topics/issues/51) as a replacement for [Issue 45](https://github.com/ansible-collections/overview/issues/45) to track changes that Ansible collection maintainers should be aware of.
+
+If you are a collection maintainer, please subscribe to this repository using the ``Watch`` button in the upper right corner of the GitHub web interface.
+
+The news are made via issues - one per announcement.
+
+# Examples of news posted via this repository
+
+- Updates made to ``ansible-test`` to detect new issues.
+- New OS distros available for testing.
+- Changes in test containers.
+- Calls for CI test matrix updates to prevent failing tests.
+- Changes/reminders about freeze and version bump dates.
+- News and required actions related to libraries' deprecation.
+- Changes to ``ansibullbot`` workflow.
```

Comment:
```suggestion
- Changes to `ansibullbot` workflow.
```
MD file, so no double backticks :)

Reply:
yeah, i know but working with rst much more always do it this way automatically:) Thanks!